root/io/io/CMakeLists.txt

```diff
@@ -6,4 +6,9 @@ RB_ADD_GBENCHMARK(IOBenchmarks
 RB_ADD_GBENCHMARK(CompressBenchmarks
   TFileBenchmarks.cxx
   LABEL short
+  LIBRARIES Core RIO ROOTDataFrame MathCore) 
+
+RB_ADD_GBENCHMARK(CompressBenchmarks2
```

Comment:
Maybe not  CompressBenchmarks2, but TFile_CompressionBenchmarks?

Reply:
I will also probably rename CompressBenchmarks as TFile_RDFSnapshot to make more clear what are you testing..

Reply:
More info about coding conventions are here: https://github.com/root-project/rootbench#simple-benchmark-template